politeiawww/user/mysql/mysql.go

```diff
@@ -437,16 +454,157 @@ func (m *mysql) UserUpdate(u user.User) error {
 		Blob:      eb,
 		UpdatedAt: time.Now().Unix(),
 	}
-	_, err = m.userDB.ExecContext(ctx,
+	_, err = tx.ExecContext(ctx,
 		"UPDATE users SET username = ?, uBlob = ?, updatedAt = ? WHERE ID = ? ",
 		ur.Username, ur.Blob, ur.UpdatedAt, ur.ID)
 	if err != nil {
 		return fmt.Errorf("create user: %v", err)
 	}
 
+	// Update user identities in multiple steps:
+	// - Get user's identities from db.
+	// - Compare against provided user.User Identities field.
```

Comment:
Currently searchs for public keys which aren't in the db yet, but it should also check for identities which are currently activated but there activated timestamp on db isn't updated yet.